make devinstall does not work

Following https://github.com/Seagate/cortx-hare#installation:

```
[vagrant@cmu hare]$ sudo make devinstall
...
--> Installing hax with 'source /data/hare/.py3venv/bin/activate; python3.6 setup.py develop --prefix /opt/seagate/cortx/hare'
Traceback (most recent call last):
  File "setup.py", line 141, in <module>
    library_dirs=[get_motr_libs_dir()],
  File "setup.py", line 97, in get_motr_libs_dir
    assert P.isfile(libmotr), f'{libmotr}: No such file'
AssertionError: /data/cortx-motr/motr/.libs/libmotr.so: No such file
make: *** [/opt/seagate/cortx/hare/lib/python3.6/site-packages/hax.egg-link] Error 1
```

Commit 571ae39

--- a/hctl
+++ b/hctl
@@ -54,6 +54,10 @@ else
         die 'Hare is not installed, please run `make devinstall`.'
 fi
 
+if [[ -d $SRC_DIR/.py3venv ]]; then
+    source "$SRC_DIR/.py3venv/bin/activate"
+fi
+
 usage() {
     # NB: `local var=$(func)` expression ignores exit status of `func`
     # and always succeeds.

